app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,30 @@
  *= require foundation_and_overrides
 
  */
+  @font-face {
+    font-family: "MKXfont";
+    src: font-url('MKXfont.ttf');
+  }
+
+  body {
+    background-image: no-repeat;
```

Comment:
Properties should be ordered background-attachment, background-image, background-image, background-position, position